hive_agent/server/routes/files.py

```diff
@@ -10,7 +10,17 @@
 
 
 BASE_DIR = "hive-agent-data/files/user"
-ALLOWED_FILE_TYPES = ["application/json", "text/csv", "text/plain", "application/pdf"]
+ALLOWED_FILE_TYPES = [
+    "application/json", 
+    "text/csv", 
+    "text/plain", 
+    "application/pdf",
+    "image/jpeg",
+    "image/jpg",            
+    "image/png",               
+    "application/msword",      
+    "application/vnd.ms-excel" 
+]
```

Comment:
Just added some more files types here.. is this what you were looking for? Lmk 🙂

Reply:
looks good